docs/contributing/get-started.md

```diff
@@ -16,6 +16,21 @@ Before raising a PR or an issue request that you read our [Contributing guide](h
 
 * Improve the [OpenFaaS CLI tooling](https://github.com/openfaas/faas-cli) with a PR or documentation
 
+### Get up to speed
+
+If you are new with Docker, Kubernetes or Go and would like to learn or just improve your skills, you may find the following links useful:
+
+* [Docker Documentation](https://docs.docker.com)
```

Comment:
I like the idea of this and the structure.

Can we find some specific topics in the docker or kube docs that would be helpful for someone or add value over something just googling "docker docs??